ci/Jenkinsfile.windows

```diff
@@ -1,4 +1,4 @@
-library 'status-jenkins-lib@v1.2.18'
+library 'status-jenkins-lib@feat/windows-installer'
```

Comment:
@borismelnik why did you merge this like this? The PR was never merged:
https://github.com/status-im/status-jenkins-lib/pull/30

And it was never tagged. I'm pretty sure I told you we'll need to merge it and tag it.

Reply:
Merged it and tagged it as `v1.2.19`.